herbert_core/utilities/general/time_waster.m

```diff
@@ -14,25 +14,30 @@
 % Input:
 % ------
 %   yin     Array of values
-%   nloop   Number of times to perform the operation
+%   nloop   Number of times to perform the operation. If nloop <= 0 then
+%           the operation is ignored.
 %
 % Output:
 % -------
 %   yout    Output array
 
 
-if nloop~=0
+if round(nloop)>0
```

Comment:
Why bother rounding? It's visual noise in the function. In matlab
```matlab
for j = 1:6.6
   j
end
% 1 2 3 4 5 6
```
If someone's silly enough to pass in a fractional number to what clearly should be an int (through name alone), that being undefined (or round down in this case) is fine.

 If you're still concerned, round it once at the beginning of the function and keep the rest of the function clean.